src/main/java/ai/elimu/model/gson/content/multimedia/AudioGson.java

```diff
@@ -1,26 +1,17 @@
 package ai.elimu.model.gson.content.multimedia;
 
 import ai.elimu.model.enums.content.AudioFormat;
+import lombok.Getter;
+import lombok.Setter;
 
 public class AudioGson extends MultimediaGson {
-    
-    private String transcription;
-    
-    private AudioFormat audioFormat;
-
-    public String getTranscription() {
-        return transcription;
-    }
 
-    public void setTranscription(String transcription) {
-        this.transcription = transcription;
-    }
+    @Getter
+    @Setter
+    private String transcription;
 
-    public AudioFormat getAudioFormat() {
-        return audioFormat;
-    }
+    @Getter
+    @Setter
+    private AudioFormat audioFormat;
```

Comment:
The webapp no longer compiles:

```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /Users/jogrimst/GitHub/webapp/src/main/java/ai/elimu/rest/v1/JavaToGsonConverter.java:[80,22] cannot find symbol
  symbol:   method setAudioType(ai.elimu.model.enums.content.AudioFormat)
  location: variable audioGson of type ai.elimu.model.gson.content.multimedia.AudioGson
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
```

Reply:
Weird, I thought did build webapp successfully before pushing. I'll have a look

Reply:
my bad, my built was using old model.
pushed the fix for this. plz check if its ok now